internal/storage/db.go

```diff
@@ -2,15 +2,20 @@ package storage
 
 import (
 	"context"
+	"database/sql"
+	"fmt"
 	"math/big"
 
 	"github.com/DIMO-Network/rewards-api/models"
 	"github.com/DIMO-Network/shared/db"
+	"github.com/ericlagergren/decimal"
 	"github.com/rs/zerolog"
 	"github.com/volatiletech/sqlboiler/v4/queries/qm"
+	"github.com/volatiletech/sqlboiler/v4/types"
 )
 
 var ether = new(big.Int).Exp(big.NewInt(10), big.NewInt(18), nil)
```

Comment:
I don't know how easy it is to use but there's a `params.Ether`

https://pkg.go.dev/github.com/ethereum/go-ethereum/params#pkg-constants